x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/arithmetic/Add.java

```diff
@@ -12,7 +12,7 @@
 /**
  * Addition function ({@code a + b}).
  */
-public class Add extends DateTimeArithmeticOperation {
+public class Add extends DateTimeArithmeticOperation implements BinaryComparisonInvertible {
```

Comment:
I've changed it to Inversible..

Since the interface is meant to convey the action intent, I find `-ble` needed. But FTR, `invertible` might still be more appropriate since "inversible" isn't really a word and a word already exists: that's why a matrix^-1 is "inverted" and so are also indexes/indices ("_inverted_ index" not "inversed index").